sig-docs/README.md

```diff
@@ -13,7 +13,7 @@ Covers documentation, doc processes, and doc publishing for Kubernetes.
 The [charter](charter.md) defines the scope and governance of the Docs Special Interest Group.
 
 ## Meetings
```

Comment:
Can you move the Americas meeting time to the first slot above the APAC meeting time?

Reply:
Just tested... It appears that the script responsible for generating the file (`make WHAT=sig-docs generate`) will arrange descriptions alphabetically (in both `sigs.yaml` as well as the `README.md`) when executed.

We could change the descriptions to something else that places the Americas meeting first.